finetune/lora.py

```diff
@@ -97,9 +125,9 @@ def main(fabric: L.Fabric, data_dir: Path, checkpoint_dir: Path, out_dir: Path):
 
     trainable_params = [p for p in model.parameters() if p.requires_grad]
     num_params = sum(p.numel() for p in trainable_params)
-    fabric.print(f"Number of trainable parameters: {num_params}")
+    fabric.print(f"Number of trainable parameters: {num_params:,}")
```

Comment:
noob q: what does this do?

Reply:
Surprised that you don't know it. I'm not flexing, just got the impression that you know everything.
It's just a neat way of displaying large numbers:
<img width="243" alt="Screenshot 2023-07-12 at 8 36 25 PM" src="https://github.com/Lightning-AI/lit-gpt/assets/58434077/684f2815-8aeb-41de-8770-4d0d7dcbf085">

Reply:
Ah nice improvement! Can you do this for all scripts that print the number of params in a separate PR?

Reply:
Sure 😄.